src/Cassandra/ConsistencyLevel.cs

```diff
@@ -76,6 +76,19 @@ public enum ConsistencyLevel
         /// <summary>
         /// Similar to <c>One</c> but only within the DC the coordinator is in.
         /// </summary>
-        LocalOne = 0x000A
+        LocalOne = 0x000A,
+        YbConsistentPrefix = One,
+        YbStrong = Quorum,
+    }
+
+    public static class ConsistencyLevelMethod
+    {
+        /// <summary>
+        /// Whether or not this consistency level corresponds to YB strong consistency.
+        /// </summary> 
+        public static bool IsStrong(this ConsistencyLevel self)
```

Comment:
Rename `self` to `level`? Also, unsure about C# syntax. Is `this` needed?

Reply:
It is extension method.
So one could use it in the following manner:
```
ConsistencyLevel level = ...
if (level.IsStrong()) { ... }
```